packages/lib-utils/src/components/details-page-header/utils/Breadcrumbs.tsx

```diff
@@ -0,0 +1,51 @@
+import { v4 as uuidv4 } from 'uuid';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export type BreadcrumbProp = { name: string; path: string; uuid?: string; id?: string };
+
+export type BreadcrumbsProps = {
+  breadcrumbs: BreadcrumbProp[];
+};
+
+export const Breadcrumbs: React.SFC<BreadcrumbsProps> = ({ breadcrumbs }) => {
+  const [crumbs, setCrumbs] = React.useState<BreadcrumbProp[]>([]);
+
+  const addUUID = (allData: BreadcrumbProp[]) => {
```

Comment:
I feel like we do something similar in the list view component... @fhlavac we may want to consider adding a utility that all components can share?